### PR TITLE
Added webpush channels to overview

### PR DESCRIPTION
**Pull request by mansimarkaur**
- [x] Add documentation.
- [ ] Add tests.
- [ ] Add a changelog entry.
- [x] Add your name in the contributors file.
- [ ] If you changed the HTTP API, update the [API_VERSION](https://github.com/Kinto/kinto/blob/master/kinto/__init__.py#L15) constant and add an API changelog entry [in the docs](https://github.com/Kinto/kinto/blob/master/docs/api/index.rst)

r? @Natim

### CODE REVIEW

File: docs/overview.rst [hunk 1]
```diff
@@ -177,27 +177,27 @@ Key features
...
+| :github:`JavaScript HTTP API client         | :github:`Python HTTP API client                         |
+| <Kinto/kinto-http.js/>`                     | <Kinto/kinto.py>`                                       |
++---------------------------------------------+---------------------------------------------------------+
+| |logo-offline|                              | |logo-admin|                                            |
+| Offline-first `JavaScript client            | :github:`Web Admin UI                                   |
+| <https://kintojs.readthedocs.io>`_          | <Kinto/kinto-admin>`                                    |
++---------------------------------------------+---------------------------------------------------------+
+| |logo-history|                              | |logo-livesync|                                         |
+| :ref:`History of changes and authorship     | Live `Push notifications                                |
+| <api-history>`                              | <https://github.com/webpush-channels/webpush-channels>`_|
```
Comment (leplatrem) [inline docs/overview.rst]: Here I would rather put a link to a Kinto plugin or documentation page. The webpush-channels project seems to be an independant project, no? (yes, using kinto.core but how does it integrate with kinto in the end?)

File: docs/overview.rst [hunk 2]
```diff
@@ -209,12 +209,6 @@ or our :ref:`tutorials <tutorials>`!
 - `kinto-http-java <https://github.com/intesens/kinto-http-java>`_: A Java HTTP Client
 - `ember-kinto <https://github.com/ptgamr/ember-kinto>`_: Offline first Ember Data adapter
 
-**Coming soon**
-
-- Push notifications using `the Push API <https://developer.mozilla.org/en-US/docs/Web/API/Push_API>`_ (:github:`under construction <Kinto/kinto-webpush>`)
-
```
Comment (leplatrem) [inline docs/overview.rst]: Ok, if webpush-channels replace kinto-webpush, then we need a documentation page that explains how to receive push notifications when some record is changed in Kinto

File: docs/overview.rst [hunk 3]
```diff
@@ -209,12 +209,6 @@ or our :ref:`tutorials <tutorials>`!
...
-(See `our roadmap <https://github.com/Kinto/kinto/wiki/Roadmap>`_)
```
Comment (leplatrem) [inline docs/overview.rst]: nitpick: I would leave the link to the roadmap here

File: docs/overview.rst (hunk 1 above)
Comment (leplatrem) [inline docs/overview.rst]: Also, please use the `:github:` shortcut like other links!

**Comment by Natim** [comment]
Closing for lack of answers. Feel free to reopen if you wish to fix the comments.

**Pull request closed without merging**